### DISCUSSION TITLE
Can't get verified because the domain for the school is incorrect.

### DISCUSSION THREAD

**Discussion by pacollins** [community/community#41457] [GitHub Education] [answered]
When applying, I receive the following errors:

> We require applicants of Lanier High School to use one of these school-issued email addresses to apply:
> - lanierhs.org
> Please select a different email address.

> We require applicants to use a school-issued email address to apply.

However, all school emails within the district are located at the following domains:

- @gcpsk12.org
- @gwinnett.k12.ga.us
- @g.gcpsk12.org

Is there a way to get this updated?

**Comment by neilhdezs** [comment]
the school is the one who should contact them to update the email domains, or contact github support, to see what solution they give you.

**Comment by samus-aran** [comment, marked as answer]
Hi @pacollins 👋 

Thank you for raising this to our attention. I've reached out to the team and they have updated the schools details. Please re-apply with the appropriate domains and let us know how you get on. 👍

**Reply by pacollins** [reply]
Thank you!

**Reply by pacollins** [reply]
@samus-aran - I am trying to renew my access and am running into the same problem again.

**Comment by akkusg** [comment]
Similar issue for Istanbul Gelisim University. Only ogr.gelisim.edu.tr is listed, which is for students. For the lecturers it is just gelisim.edu.tr.
Can you fix that?
